README.md

```diff
@@ -42,19 +42,19 @@ From the python shell, start up the db server in the background:
 ```sh
 sh scripts/start_server.sh &
 ```
-### Using the DJORNL Exascale data parser
+### using the djornl exascale data parser
 
-Use the parser to load some data; if there are no errors in the data, you will get output detailing the number of nodes and edges added to the database. The shell environment variable `RES_ROOT_DATA_PATH` contains the path to the data directory containing `manifest.yaml` and the data files.
+use the parser to load some data; if there are no errors in the data, you will get output detailing the number of nodes and edges added to the database. the shell environment variable `res_root_data_path` contains the path to the data directory containing `manifest.yaml` and the data files.
 
-By default, `RES_ROOT_DATA_PATH` is set to point to the exascale data; that dataset can be loaded into the db with the command:
+by default, `res_root_data_path` is set to point to the exascale data; that dataset can be loaded into the db with the command:
 ```sh
 python -m importers.djornl.parser
 ```
 
-In this example, we load a small test dataset:
+in this example, we load a small test dataset:
 
 ```sh
-RES_ROOT_DATA_PATH=/app/spec/test/djornl/test_data python -m importers.djornl.parser
+res_root_data_path=/app/spec/test/djornl/test_data python -m importers.djornl.parser
```

Comment:
I think it makes sense to leave `RES_ROOT_DATA_PATH` as uppercase because it represents an environment variable, and this is traditional. It is not wrong to make it lowercase, but if it is changed to lowercase here then the casing should be consistent. In particular, it should also be changed to lowercase in the edits below and any code that references it. My personal preference is to leave it uppercase to convey its status as an environment variable and to limit the scope of this PR.